src/Fantomas.Tests/CompilerDirectivesTests.fs

```diff
@@ -2659,3 +2659,58 @@ let ``indented #if directive inside another non-indented #if directive should fo
 #endif
 #endif
 """
+
+[<Test>]
+let ``double try-with, inner #if directive should not throw error, 1969`` () =
+    let correctlyFormatedSrc = 
```

Comment:
Please use the unit test template as listed in the [contribution guidelines](https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#pull-request-ground-rules).
Even if the input and output are the same.
Just for consistency sake.

Reply:
Yes I'll do that as soon as possible 👍

Reply:
Got it. I changed that.

However, I'm not sure how often that patterns `fantomas(src) = src` occur in tests but maybe it would be beneficial to introduce some "test helper" method that would simplify that kind of 'identity tests'? 🤔 I'm just loudly thinking, because maybe here it just doesn't make sense...

Reply:
There is a helper function for this, but I do not recommend it.
It can be difficult to reason about these things when the style guide changes.
So, in short, Fantomas follows https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting.
If this ever changes (which spoiler it often does), then your source input should not change and only the output needs to be updated accordingly.

Reply:
It make sense. Thanks for the explanation. I didn't think about scenario when style guide change :)